crates/beerus-rpc/src/api.rs

```diff
@@ -85,16 +85,11 @@ pub trait BeerusApi {
     async fn get_transaction_by_hash(&self, tx_hash: &str) -> Result<Transaction, Error>;
 
     #[method(name = "getBlockTransactionCount")]
-    async fn get_block_transaction_count(
-        &self,
-        block_id_type: String,
-        block_id: String,
-    ) -> Result<u64, Error>;
+    async fn get_block_transaction_count(&self, block_id: String) -> Result<u64, Error>;
```

Comment:
Valid for all other instances: Shouldn't we use BlockId as a type in the method definition params?